exercises/3-revenge-of-the-automated-testing/README.md

```diff
@@ -415,393 +462,402 @@ git push
+
+   ```bash
+   git checkout develop
+   ```
+
+   ```bash
+   git merge feature/important-flag
+   ```
+
+   When the editor screen appears in the terminal after running the merge, type `:q` and hit enter to quit the editor.
+
+   ```bash
+   git push --all
+   ```
 
-<kbd>📝 todolist/src/components/TodoItem.vue</kbd>
-```html
-    <!-- TODO - SVG for use in Exercise3 -->
-    <md-button class="important-flag" @click="markImportant()">
-        <svg :class="{'red-flag': todoItem.important}"  height="24" viewBox="0 0 24 24" width="24" xmlns="http://www.w3.org/2000/svg" ><path d="M0 0h24v24H0z" fill="none"/><path d="M14.4 6L14 4H5v17h2v-7h5.6l.4 2h7V6z"/></svg>
-    </md-button>
-```
+#### 2c - Create todolist e2e tests
 
-15. Finally - we need to make it so that when a new todo item is created it will have an important property. Head to `src/store/actions.js` and add `important: false`  below `completed: false` in the `addTodo(){}` action.
+Using [Nightwatch.js](http://nightwatchjs.org/) We will write a simple e2e test to test the functionality of the feature we just implemented.
 
-![fe-add-actions-important](../images/exercise3/fe-add-actions-important.jpg)
+1.  First, we need to create an e2e test file.
 
+    ```bash
+    touch tests/e2e/specs/importantFlag.js
+    ```
 
-16. The previously failing tests should have started to pass now. With this work done, let's commit our code. On the terminal, run
-```bash
-git add .
-```
-```bash
-git commit -m "Implementing the todoitem flag"
-```
-```bash
-git push
-```
+1.  Open this new file in your code editor and create an initial template for an e2e test as shown below:
 
-17. Start our local development server (if not already running) to manually verify the changes 
+    <kbd>📝 todolist/tests/e2e/specs/importantFlag.js</kbd>
 
-```bash
-npm run serve:all
-```
-
-18. Open our local todolist app (http://localhost:8080/#/todo). If we try to use our important flag, we should see it's still not behaving as expected; this is because we're not updating the state of the app in response to the click event.
+    ```javascript
+    module.exports = {
+      "Testing important flag setting": (browser) => {},
+    };
+    ```
 
-19. We need to implement the `actions` and `mutations` for our feature. Let's start with the tests. Open the `tests/unit/javascript/actions.spec.js` and navigate to the bottom of the file. Our action should should commit the `MARK_TODO_IMPORTANT` to the mutations. Scroll to the end of the test file and implement the skeleton test by adding `expect(commit.firstCall.args[0]).toBe("MARK_TODO_IMPORTANT");` as the assertion.
+    ```javascript
+    module.exports = {
+      "Testing important flag setting': browser => {
 
-<kbd>📝 todolist/tests/unit/javascript/actions.spec.js</kbd>
-```javascript
-  it("should call MARK_TODO_IMPORTANT", done => {
```

Comment:
All tests in this section must be added to `todolist` before this PR can be merged